operator/v1/types_dns.go

```diff
@@ -70,6 +70,19 @@ type DNSSpec struct {
 	// DNS
 	// +optional
 	ManagementState ManagementState `json:"managementState,omitempty"`
+
+	// operatorLogLevel controls the logging level of the DNS Operator.
+	// +optional
+	OperatorLogLevel LogLevel `json:"operatorLogLevel,omitempty"`
+
+	// logLevel describes the logging verbosity of the DNSController for CoreDNS.
+	// Enables the CLASSES(https://github.com/coredns/coredns/tree/master/plugin/log#syntax)
+	// of coredns w.r.t to the LogLevel defined in openshift api.
+	// logLevel "Normal"  will enable errors class as we enable errors plugin by default log . { class error }
+	// logLevel "Debug" will enable log . { class denial error }
+	// logLevel "Trace" will enable  log . { class all }
```

Comment:
This will look pretty messy in the `oc explain` output, and the user shouldn't need to worry about implementation details.  How about this:
```suggestion
	// logLevel describes the desired logging verbosity for CoreDNS.
	// Any one of the following values may be specified:
	// * Normal logs errors from upstream resolvers.
	// * Debug logs errors, NXDOMAIN responses, and NODATA responses.
	// * Trace logs errors and all responses.
```